### PR TITLE
Decode info dict with jsonpickle

### PR DESCRIPTION
**Pull request by vnmabus**
This allows to recover complex objects, such as numpy arrays, stored in the info dict.

### CODE REVIEW

File: incense/experiment.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
 # -*- coding: future_fstrings -*-
+import json
 from typing import *
 
-import pandas as pd
+import jsonpickle
```
[the remark this answers is not in the document]
Comment (vnmabus) [reply incense/experiment.py]: Added in 9c3c54b89ef5d1e9c62834583e8f658e1d23a15e.

File: incense/experiment.py [hunk 2]
```diff
@@ -28,10 +31,11 @@ def __getattr__(self, item):
     @classmethod
     def from_db_object(cls, database, grid_filesystem, experiment_data: dict, loader):
         data = freeze(experiment_data)
+        info = freeze(jsonpickle.loads(json.dumps(experiment_data['info'])))
```
[the remark this answers is not in the document]
Comment (vnmabus) [reply incense/experiment.py]: Changed in 9c3c54b89ef5d1e9c62834583e8f658e1d23a15e.

**Comment by vnmabus** [comment]
> Thanks for making the first PR to incense!
> It probably would make sense to give the experiment loader a flag `unpickle` with a default to toggle the behavior in case one does not have the pickled classes available or does not care to unpickle the instances.

Added in 9c3c54b89ef5d1e9c62834583e8f658e1d23a15e.

> Also please add a simple unittest.

I do not know how I am supposed to run the tests. The tests do not run at all if I use `python setup.py test`. If I try to run them using `pytest` most tests fail because there is no database with the required experiment results.

**Comment by JarnoRFB** [comment]
Thanks, the changes look good!

> I do not know how I am supposed to run the tests. The tests do not run at all if I use python setup.py test. If I try to run them using pytest most tests fail because there is no database with the required experiment results.

Probably should have added some more instructions for this. You can either use a devcontainer from vscode as described in the readme or use https://github.com/JarnoRFB/incense/blob/master/infrastructure/sacred_setup/docker-compose.yml . In any case you will need to have docker installed. Once the containers are running you need to populate the database with `python tests/example_experiment/conduct.py`. You could modify the code there to add an experiment with some objects in its info_dict, you could also create an experiment in a test  as is also done several times in the unit tests, e.g. https://github.com/JarnoRFB/incense/blob/master/tests/test_experiment_loader.py#L49 . If want to start with a clean database again, use the `delete_db.py` script.

**Comment by vnmabus** [comment]
Test added.

**Pull request merged**